test/functional/test-url-replacements.js

```diff
@@ -166,6 +169,7 @@ describes.sandboxed('UrlReplacements', {}, () => {
         documentElement: {
           nodeType: /* element */ 1,
         },
+        head: document.createElement('head'),
```

Comment:
is this needed?

Reply:
if we don't include it this will break https://github.com/ampproject/amphtml/pull/19844/files#diff-775f4cb823bb027213fbff02cfb81778R605

Reply:
This was removed with change back to `parseUrlDeprecated`